### PR TITLE
Clarify JS API docs (IP-9077)

### PR DESCRIPTION
**Pull request by terrymun**
This PR clears up the possible confusion with the use of placeholder values.

### CODE REVIEW

File: pages/flipbook-javascript-api--migration-guide.md [hunk 1]
```diff
@@ -24,6 +22,8 @@ r.id=a;r.async=1;r.src=p+'/'+e+'.js'})
 <!-- End of async iPaper Flipbook API script -->
 ```
 
+{% include note.html content="Both `<ApiBaseUrl>` and `<YourApiKey>` in the snippet above will be substituted with partner and license-dependent configurations and be automatically populated, when you retrieve the API script from the admin." %}
```
Comment (itniels) [inline pages/flipbook-javascript-api--migration-guide.md:25]: `<YourApiKey>`Will not be auto populated, the user needs to login to their license and go to Settings ->APi Keys, where they will need to create one if none exists.

File: pages/flipbook-javascript-api--quick-start.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ tags: Flipbook JS API
...
 (function(i,P,a,p,e,r){if(i.getElementById(a=a+'-'+e))return;
 r=i.querySelector(P).parentNode.appendChild(i.createElement(P));
 r.id=a;r.async=1;r.src=p+'/'+e+'.js'})
-(document,'script','ipaper-api','<FlipbookApiBaseUrl>','<YourFlipbookApiKey>');
+(document,'script','ipaper-api','<ApiBaseUrl>','<YourApiKey>');
 </script>
 <!-- End of async iPaper Flipbook API script -->
 ```
 
+{% include note.html content="Both `<ApiBaseUrl>` and `<YourApiKey>` in the snippet above will be substituted with partner and license-dependent configurations and be automatically populated, when you retrieve the API script from the admin." %}
```
Comment (itniels) [inline pages/flipbook-javascript-api--quick-start.md:27]: `<YourApiKey>`Will not be auto populated, the user needs to login to their license and go to Settings ->APi Keys, where they will need to create one if none exists. :)

File: pages/display-javascript-api--quick-start.md [hunk 3]
```diff
@@ -23,15 +23,17 @@ The Display Javascript API [exposes magic commands](/display-javascript-api/comm
 
 ## Setup
 
-A Display instance and its CTA button can only be shown on pages that has the Display JavaScript API embedded. The Display API embed script will look something like this, with `<YourDisplayApiKey>` being substitued with partner and license-dependent configurations.
+A Display instance and its CTA button can only be shown on pages that has the Display JavaScript API embedded. The Display API embed script will look something like this, with `<YourApiKey>` being substitued with partner and license-dependent configurations:
```
Comment (itniels) [inline pages/display-javascript-api--quick-start.md:26]:
```suggestion
A Display instance and its CTA button can only be shown on pages that has the Display JavaScript API embedded. The Display API embed script will look something like this, with `<YourApiKey>` being substituted with your license API key:
```

**Comment by terrymun** [comment]
@itniels The so-called API key (or actually, just a secret base64 encoded string), is not the API key that is created for the license. Perhaps I should then rename it since it's causing so much confusion.

File: pages/flipbook-javascript-api--migration-guide.md (hunk 1 above)
Comment (terrymun) [reply pages/flipbook-javascript-api--migration-guide.md:25]: It is... 
flipbook-api-script

File: pages/flipbook-javascript-api--quick-start.md (hunk 2 above)
Comment (terrymun) [reply pages/flipbook-javascript-api--quick-start.md:27]: It is auto populated: 
display-api-script

**Comment by itniels** [comment]
@terrymun Ahh!! yes, please do rename it! i had it for sure to be the license API key (just from looking at it!) 😞

**Comment by terrymun** [comment]
> @terrymun Ahh!! yes, please do rename it! i had it for sure to be the license API key (just from looking at it!) 😞

What naming would you suggest? I am thinking of `YourApiSecret`.

**Comment by itniels** [comment]
> > @terrymun Ahh!! yes, please do rename it! i had it for sure to be the license API key (just from looking at it!) 😞
> 
> What naming would you suggest? I am thinking of `YourApiSecret`.

How about: `<ApiSecret>` just like `ApiBaseUrl` without the "your" to confuse, as that sounds like something i should have on hand 🙂

**Comment by terrymun** [comment]
Good idea, I'll run with that then.

**Pull request merged**